src/sqlfluff/core/rules/std/L036.py

```diff
@@ -1,10 +1,23 @@
 """Implementation of Rule L036."""
 
-from collections import namedtuple
+from typing import List, NamedTuple
 
-from sqlfluff.core.rules.base import BaseCrawler, LintResult
+from sqlfluff.core.parser import BaseSegment
+from sqlfluff.core.rules.base import BaseCrawler, LintFix, LintResult
+from sqlfluff.core.rules.doc_decorators import document_fix_compatible
 
 
+class SelectTargetsInfo(NamedTuple):
```

Comment:
Refactored a `collections.namedtuple` to a nicer `typing.NamedTuple`, also made a change or two.